cmd/nitro/nitro.go

```diff
@@ -638,6 +638,38 @@ func mainImpl() int {
 			}
 		}
 	}
+
+	// Before starting the node, wait until the transaction that deployed rollup is finalized
+	if nodeConfig.Node.ParentChainReader.Enable && rollupAddrs.DeployedAt > 0 {
```

Comment:
I want an option to disable that wait.
I think we also need to register to sigint before this loop and do an orderly shutdown if we got it.

Reply:
fixed it